README.md

```diff
@@ -17,21 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] String
-[] Boolean
-[] Undefined
-[] NaN
-[] Number
-[] Array
-[] Null
+[x] String
+[x] Boolean
+[x] Undefined
+[x] NaN
+[x] Number
+[x] Array
+[x] Null
 ```
```

Comment:
The correct answers are `string` `boolean` `undefined` `null` `number`. Array is often referred to as a "data type", but it's really a data collection (i.e., it can contain the more basic or "primitive" data types).

`NaN` is not a data type. It is, counter-intuitively, a type of `number`.